src/site/layouts/health_care_local_facility.drupal.liquid

```diff
@@ -211,7 +211,7 @@ more.\r\n",
 {% include "src/site/includes/header.html" with drupalTags = true %}
 {% include "src/site/includes/alerts.drupal.liquid" %}
 {% include "src/site/includes/preview-edit.drupal.liquid" %}
-{% include "src/site/includes/breadcrumbs.drupal.liquid" with hideHomeBreadcrumb = true %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" with hideHomeBreadcrumb = true deriveBreadcrumbsFromUrl = true %}
```

Comment:
I don't know how other CMS-backed sites do breadcrumbs but man it feels like ours are really complicated